apps/angular/src/app/features/search-form/search-form.component.html

```diff
@@ -0,0 +1,9 @@
+<form class="search-form">
+	<mat-form-field appearance="outline" class="search-field">
+		<mat-label class="search-title">Search by title</mat-label>
+		<input matInput placeholder="One Piece" type="search" class="search-input" [formControl]="searchControl">
+	</mat-form-field>
+	<button mat-fab type="submit" class="search-button" (click)="onSubmit($event)">
```

Comment:
And then remove this redundant handler
```suggestion
	<button mat-fab type="submit" class="search-button">
```